projects/core/src/styles/theme.dark.scss

```diff
@@ -16,6 +16,25 @@
   --cds-global-typography-link-color-hover: #{$cds-global-color-blue-200};
   --cds-global-typography-link-color-visited: #{$cds-global-color-lavender-300};
   --cds-global-typography-link-color-visited-hover: #{$cds-global-color-lavender-200};
+  --cds-global-typography-color-450: #{$cds-global-color-construction-50};
```

Comment:
These were added as part of #265 so any `-global-typography changes in here don't affect the existing colors outside of the ones that were recently added for the ng v16 release.